__mocks__/passportStatusesMock.json

```diff
@@ -36,7 +36,7 @@
         "lastModifiedDate": "2022-10-24T10:48:00.596207Z",
         "_links": {
           "search": {
-            "href": "https://passport-status-api.dev.dev-rhp.dts-stn.com/api/v1/passport-statuses/_search?dateOfBirth=1991-01-07&fileNumber=000672BC&firstName=Maxence&lastName=Fourni%C3%A9r&unique=true"
+            "href": "https://passport-status-api.dev.dev-rhp.dts-stn.com/api/v1/passport-statuses/_search?dateOfBirth=1991-01-07&fileNumber=000672BC&givenName=Maxence&surname=Fourni%C3%A9r&unique=true"
```

Comment:
I believe you can replace the URL-escaped characters in the surname with actual accented characters (for enhanced readability).

Reply:
That's no excuse!
(resolving)